name.abuchen.portfolio/src/name/abuchen/portfolio/messages.properties

```diff
@@ -574,6 +574,8 @@ MsgUnsupportedVersionClientFiled = This file was created with a newer version of
 
 MsgXMLFormatInvalid = Invalid XML Format: {0}
 
+PDFLabelShares = Shares
```

Comment:
There is already a label "shares" - ```CSVColumn_Shares```. I propose instead of duplicating the translation terms, just use the existing label. It is easier for the translators than checking the same thing over again.

(I understand there are duplicate labels between the two packages "portfolio" and "portfolio.ui". It is not so easy to deduplicate them therefore I accepted that. But within on package, let's not add duplications).

Reply:
Also finished.